### PR TITLE
Add troubleshooting note about partial continuous aggregates

### PR DESCRIPTION
**Pull request by JamesGuthrie**
# Description

Partial continuous aggregates cannot be migrated across PostgreSQL versions.

### CODE REVIEW

**Review by arajkumar — APPROVED** [review]

File: migrate/troubleshooting.md [hunk 1]
```diff
@@ -221,3 +221,25 @@ Timescale instance or "merge" source databases to target schemas.
...
+
+```SQL
+SELECT exists (SELECT 1 FROM timescaledb_information.continuous_aggregates WHERE NOT finalized);
+```
+
+If you do happen to migrate partial continuous aggregates across PostgreSQL
+versions, the following error appears when the continuous aggregate is queried:
+`ERROR:  insufficient data left in message`.
+
+[migrate]: /migrate/:currentVersion:/live-migration/
```
Comment (billy-the-fish) [inline migrate/troubleshooting.md:245]:
```suggestion

## Migrate partial continuous aggregates

In order to the improve performance and compatibility of continuous aggregates, 
[TimescaleDB v2.7][release-270] replaces _partial_ continuous aggregate with 
_finalized_ continuous aggregates.

If you have partial continuous aggregates in your database and intend to upgrade your
 PostgreSQL installation, you must first [migrate] from partial to finalized continuous
 aggregates. It is not possible to migrate partial continuous across PostgreSQL versions. 

To test your database for partial continuous aggregates, run the following query:

```SQL
SELECT exists (SELECT 1 FROM timescaledb_information.continuous_aggregates WHERE NOT finalized);
```

If you accidentally migrate partial continuous aggregates across PostgreSQL
versions, you see the following error when you query any continuous:

```
ERROR:  insufficient data left in message`.
```

[migrate]: /migrate/:currentVersion:/live-migration/
[release-270]: /about/:currentVersion:/release-notes/past-releases/#270-2022-05-24

```

Comment (billy-the-fish) [inline migrate/troubleshooting.md:245]:
```suggestion

## Migrate partial continuous aggregates

In order to the improve performance and compatibility of continuous aggregates, 
[TimescaleDB v2.7][release-270] replaces _partial_ continuous aggregate with 
_finalized_ continuous aggregates.

If you have partial continuous aggregates in your database and intend to upgrade your
 PostgreSQL installation, you must first [migrate] from partial to finalized continuous
 aggregates. It is not possible to migrate partial continuous across PostgreSQL versions. 

To test your database for partial continuous aggregates, run the following query:

```SQL
SELECT exists (SELECT 1 FROM timescaledb_information.continuous_aggregates WHERE NOT finalized);
```

If you accidentally migrate partial continuous aggregates across PostgreSQL
versions, you see the following error when you query any continuous:

```
ERROR:  insufficient data left in message`.
```

[migrate]: /migrate/:currentVersion:/live-migration/
[release-270]: /about/:currentVersion:/release-notes/past-releases/#270-2022-05-24




```

**Review by billy-the-fish — APPROVED** [review]
LGTM

File: migrate/troubleshooting.md [hunk 2]
```diff
@@ -221,3 +221,29 @@ Timescale instance or "merge" source databases to target schemas.
 The `tsdbadmin` database user is the most powerful available on Timescale, but it
 is not a true superuser. Review your application for use of superuser privileged
 operations and mitigate before migrating.
+
+## Migrate partial continuous aggregates
+
+In order to improve the performance and compatibility of continuous aggregates, 
+[TimescaleDB v2.7][release-270] replaces _partial_ continuous aggregate with
```
Comment (JamesGuthrie) [inline migrate/troubleshooting.md]:
```suggestion
[TimescaleDB v2.7][release-270] replaces _partial_ continuous aggregates with 
```

File: migrate/troubleshooting.md [hunk 3]
```diff
@@ -221,3 +221,29 @@ Timescale instance or "merge" source databases to target schemas.
...
+_finalized_ continuous aggregates.
+
+To test your database for partial continuous aggregates, run the following query:
+
+```SQL
+SELECT exists (SELECT 1 FROM timescaledb_information.continuous_aggregates WHERE NOT finalized);
+```
+
+If you have partial continuous aggregates in your database,  [migrate your existing database][migrate] 
+from partial to finalized continuous aggregates before you upgrade your PostgreSQL installation
```
Comment (JamesGuthrie) [inline migrate/troubleshooting.md]:
```suggestion
If you have partial continuous aggregates in your database, [convert them][migrate] 
from partial to finalized before you migrate your database.
```

File: migrate/troubleshooting.md [hunk 4]
```diff
@@ -221,3 +221,29 @@ Timescale instance or "merge" source databases to target schemas.
...
+```
+
+If you have partial continuous aggregates in your database,  [migrate your existing database][migrate] 
+from partial to finalized continuous aggregates before you upgrade your PostgreSQL installation 
+
+If you accidentally migrate partial continuous aggregates across PostgreSQL
+versions, you see the following error when you query any continuous aggregates:
+
+```
+ERROR:  insufficient data left in message`.
```
Comment (JamesGuthrie) [inline migrate/troubleshooting.md]:
```suggestion
ERROR:  insufficient data left in message.
```

File: migrate/troubleshooting.md [hunk 5]
```diff
@@ -221,3 +221,29 @@ Timescale instance or "merge" source databases to target schemas.
...
+[TimescaleDB v2.7][release-270] replaces _partial_ continuous aggregates with 
+_finalized_ continuous aggregates.
+
+To test your database for partial continuous aggregates, run the following query:
+
+```SQL
+SELECT exists (SELECT 1 FROM timescaledb_information.continuous_aggregates WHERE NOT finalized);
+```
+
+If you have partial continuous aggregates in your database, [convert them][migrate]
```
Comment (JamesGuthrie) [inline migrate/troubleshooting.md]:
```suggestion
If you have partial continuous aggregates in your database, [migrate them][migrate] 
```

**Pull request merged**